src/gui/dialogues/firstrundialog.cpp

```diff
@@ -120,8 +120,6 @@ bool FirstRunDialog::finish()
         db_fail_msg_box.exec();
         return false;
     }
-    aDB()->disconnect(); // reset db connection to refresh layout after initial setup.
-    aDB()->connect();
```

Comment:
Great that this got decoupled!

Reply:
that's the benefit of having up-to date layout data around, no need to re-call conect() or update()